modules/shell/command_special_test.go

```diff
@@ -0,0 +1,47 @@
+// +build special
+
+package shell
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
```

Comment:
Looks like this violated the rules in `goimports`. Specifically, thirdparty libs should be listed separately from stdlibs. Can you update this to be:

```
import (
        "bytes"
        "fmt"
        "testing"

        "github.com/stretchr/testify/assert"
)
```

Reply:
Thought my vim plugin does that automatically, but apparently I did not configure this :-)